### PR TITLE
Postman

### PR DESCRIPTION
**Pull request by Drapegnik**
* add postman configs
* update docs

### CODE REVIEW

File: postman/babajka.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		"description": "babajka api",
+		"schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Users",
+			"description": "",
+			"item": [
+				{
+					"name": "Usets: get all",
```
Comment (tyndria) [inline postman/babajka.postman_collection.json]: You mean, "Users: get all"?

Comment (Drapegnik) [inline postman/babajka.postman_collection.json]: yes, but in can be later fixed from postman app

Comment (tyndria) [inline postman/babajka.postman_collection.json]: Ok, just said :)

**Comment by UladBohdan** [comment]
works for me👍
the only thing is I would rename environment to "babajka-dev"

**Comment by UladBohdan** [comment]
require changes in port configuration in case #4 is pushed before this PR.

**Comment by Drapegnik** [comment]
[setup postman](https://trello.com/c/rYBl4Tfj/17-setup-postman)

**Pull request merged**